src/vma/dev/net_device_table_mgr.h

```diff
@@ -135,6 +135,7 @@ class net_device_table_mgr : public cache_table_mgr<ip_address, net_device_val*>
 	bool 			verify_ipoib_mode(struct ifaddrs* ifa);
 	bool			verify_mlx4_ib_device(const char* ifname);
 	bool 			verify_eth_qp_creation(const char* ifname);
+	bool 			check_preconditions(const char* interface_name, struct ifaddrs * ifa);
```

Comment:
All methods are verify_XXX.
Therefore need to rename check_preconditions() to verify_preconditions()